bin/check-a11y.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+import { readFileSync } from 'fs'
+import chalk from 'chalk'
+
+import AxeBuilder from '@axe-core/webdriverjs'
+import { Builder, By, until } from 'selenium-webdriver'
```

Comment:
I'm not used to work with Selenium and I don't know what the rest of the team think, but IMO another solution like Cypress could be more "friendly": https://timdeschryver.dev/blog/setting-up-cypress-with-axe-for-accessibility. Could you try it?

Reply:
Not sure to understand the "friendly"... Could you be more explicit?

Reply:
Well, maybe this can help to understand what I tried to mean with "friendly": https://2022.stateofjs.com/en-US/libraries/#tier_list